src/main/java/macrobase/runtime/MacroBaseServer.java

```diff
@@ -30,6 +31,7 @@ public String getName() {
     public void initialize(Bootstrap<MacroBaseConf> bootstrap) {
         bootstrap.addCommand(new MacroBaseBatchCommand());
         bootstrap.addCommand(new MacroBaseStreamingCommand());
+        bootstrap.addCommand(new MacroBaseContextualBatchCommand());
```

Comment:
This shouldn't be necessary: https://github.com/stanford-futuredata/macrobase/blob/master/src/main/java/macrobase/runtime/MacroBaseServer.java#L30